tests/test_ffmpeg_utils_unit.py

```diff
@@ -5,14 +5,14 @@
 
 
 def test_ffmpeg_seq_frame_vid_smokescreen() -> None:
-    """"""
+    """ """
```

Comment:
Same comment on empty docstrings here and below as well? Do you think we can add a quick 1-line docstring for them?